scripts/bootstrap.sh.in

```diff
@@ -68,6 +68,8 @@ pillar_roots:
 use_superseded:
   - module.run
 EOF
+
+        ${SALT_CALL} --local --retcode-passthrough saltutil.sync_all saltenv=metalk8s-@VERSION@
```

Comment:
we start seeing this kind if calls very often, should we create a wrapper not to rewrite the common arguments every time ?

Reply:
There's #626